training_config/multilang_dependency_parser.jsonnet

```diff
@@ -0,0 +1,120 @@
+// Configuration for the multi-lingual dependency parser model based on:
+// Schuster et al. "Cross-Lingual Alignment of Contextual Word Embeddings, with Applications to Zero-shot Dependency Parsing"
+// https://arxiv.org/abs/1902.09492 (NAACL 2019)
+//
+// To recompute alignemts for ELMo, refer to: https://github.com/TalSchuster/CrossLingualELMo
+// For the dataset, refer to https://github.com/ryanmcd/uni-dep-tb
+{
+    "dataset_reader": {
+        "type": "universal_dependencies_multilang",
+        "languages": ["en", "de", "it", "fr", "pt", "sv"],
+        "alternate": true,
+        "instances_per_file": 32,
+        "is_first_pass_for_vocab": true,
+        "lazy": true,
+        "token_indexers": {
+            "elmo": {
+                "type": "elmo_characters"
+            }
+        },
+        "use_language_specific_pos": false
+    },
+    "iterator": {
+        "type": "same_lang",
+        "batch_size": 32,
+        "instances_per_epoch": 32000
+    },
+    "model": {
+        "type": "biaffine_parser_multilang",
+        "arc_representation_dim": 500,
+        "dropout": 0.33,
+        "encoder": {
+            "type": "lstm",
+            "bidirectional": true,
+            "dropout": 0.33,
+            "hidden_size": 200,
+            "input_size": 1074,
+            "num_layers": 3
+        },
+        "langs_for_early_stop": [
+            "en",
+            "de",
+            "it",
+            "fr",
+            "pt",
+            "sv"
+        ],
+        "pos_tag_embedding": {
+            "embedding_dim": 50,
+            "vocab_namespace": "pos"
+        },
+        "tag_representation_dim": 100,
+        "text_field_embedder": {
+            "token_embedders": {
+                "elmo": {
+                    "type": "elmo_token_embedder_multilang",
+                    "aligning_files": {
+                        "en": "https://www.dropbox.com/s/nufj4pxxgv5838r/en_best_mapping.pth?dl=1",
```

Comment:
Do you mind if we host these on S3, and change the paths to point there?  If this is going to be in the library, it'd be best we if can control the dependencies here.  It'd also be great if you have a trained model that we can host (and add to our demo, but not in this PR).

Reply:
Sure thing! please do.
I have a trained model that is based on a slightly older version of the repo. After we merge this PR I can quickly train a model and share.

Reply:
Great, I will copy these over some time today or tomorrow, and let you know what paths to change these to.

Reply:
Ok, this is done.  All of these files should be available at `https://s3-us-west-2.amazonaws.com/allennlp/models/multilingual_elmo/`.

Reply:
Thanks! I might get to this and to implement the tests only in a few days.